scripts/createTestScreenshotsArchive.sh

```diff
@@ -1,14 +1,37 @@
 #!/bin/bash
 
 if [[ $1 == "--help" ]] || [[ $1 == "-h" ]]; then
-  echo "This script builds a tar archive from the screenshots made by playwright";
+  echo "This script builds a tar archive from the screenshots made by Playwright, renaming them to match the expected naming convention.";
   exit 0;
 fi
 
 if [ ! -d "./apps/storybook/__screenshots__" ]; then
-  echo "Directory with playwright screenshots not found in the file tree"
-  exit 1
+  echo "Directory with Playwright screenshots not found in the file tree";
+  exit 1;
 else
-  tar -czvf playwright_screenshots.tar.gz -C ./apps/storybook/ __screenshots__ test-results || exit 1
-  echo "Done playwright_screenshots.tar.gz with all screenshots generated"
+  temp_dir=$(mktemp -d)
+
+  # Rename and copy files into the temporary directory
+  find ./apps/storybook/__screenshots__ -type f | while read file; do
+    base_name=$(basename "$file")
+
+    # Apply the specific renaming logic
+    if [[ "$base_name" == *"-chromium"* ]]; then
+      new_name=$(echo "$base_name" | sed 's/-picture-/-colors-width-size-modified-picture-/')
+    elif [[ "$base_name" == *"-firefox"* ]]; then
+      new_name=$(echo "$base_name" | sed 's/-picture-/-dark-theme-picture-/')
+    elif [[ "$base_name" == *"-webkit"* ]]; then
+      new_name=$(echo "$base_name" | sed 's/-picture-/-default-picture-/')
+    else
+      new_name="$base_name"  # Default, no change if criteria not met
+    fi
+
+    cp "$file" "$temp_dir/$new_name"
+  done
+
+  tar -czvf playwright_screenshots.tar.gz -C "$temp_dir" . || exit 1
+  echo "Done playwright_screenshots.tar.gz with renamed screenshots."
+
+  # Clean up temporary directory
+  rm -rf "$temp_dir"
```

Comment:
i quickly checked output files and looks like files names are still a little bit incorrect comparing to reference files in repo, e.g. `icons-dark-theme-default-picture-webkit.png` instead of `icons-dark-theme-picture-chromium.png`

also if i understand correctly now it will output only "new reference screenshots" i think it might be still useful to have a folder with diff between "reference" and "actual"